templates/index.html

```diff
@@ -39,19 +39,22 @@
 				<div class="row">
 					<div class="col-auto">
 						<h1 class="border top-row">Who's In?</h1>
+						<h4 class="border top-row">Named devices</h4>
 					</div>
-				{% if harmony_domain %}
+					{% if note['note'] %}
```

Comment:
I'm curious why `if note['note']` is used here to check the existence of notes, whilst later on `if note` is used.

Reply:
This is so it only shows a note if the note is not empty. Otherwise empty notes would be displayed. Later on I don't care if it's empty or not.